### DISCUSSION TITLE
猴子补丁如何使用？

### DISCUSSION THREAD

**Discussion by Hanx152** [nonebot/discussions#126] [问答]
本人新手，按FAQ里说的把那段代码放到nonebot.run()之前没法用，是我没理解对意思吗

**Comment by mnixry** [comment]
非常遗憾, 这段代码由于uvicorn的更新目前已经无法使用. 不过对于`2.0.0b1`以及更高版本, 你只需要禁用代码修改重载就应该使用的是能够进行子进程调用的事件循环

**Reply by Hanx152** [reply]
请问能告诉我一下具体的操作吗，我不是很会这个
